routes/index.js

```diff
@@ -39,28 +40,29 @@ router.get('/updateschema', passportFunction.adminLoggedIn, function(req, res, n
     //     ],
     //     private: false
     // }]
-    var stanza = [];
-    var newLyric = [];
     Song.find(function(err, songs) {
         songs.forEach((song, i, arr) => {
-            song.lyric.forEach((line) => {
-                if (line !== '') {
-                    stanza.push(line)
-                } else {
-                    newLyric.push(stanza)
-                    stanza = [];
-                }
-            })
-            newLyric.push(stanza)
-            stanza = [];
-            song.lyric = newLyric;
-            delete song.private
-            song.save(function(err) {
-                if (err) next(err)
-                if (i === arr.length - 1) {
-                    res.send('migration success')
-                }
-            })
+            song.temp = 'ahahhaa'
```

Comment:
yes im just testing whether mongodb can have the additional field on its schema without explicitly updating the schema